base/doc/fntguide.tex

```diff
@@ -489,14 +504,26 @@ \subsection{Parameters for author commands}
 and the |ssub| size function, see Section~\ref{sec:sizefunct}.
 
 \begin{decl}
-  |\itdefault| \\
-  |\sldefault| \\
-  |\scdefault| \\
+  |\itdefault|  \\
+  |\sldefault|  \\
+  |\scdefault|  \\
+  |\sscdefault| \\
+  |\swdefault|  \\
+  |\ulcdefault| \\
   |\updefault|
 \end{decl}
+\NEWfeature{2020/02/02}
 The shapes selected by |\textit|, |\itshape|, |\textsl|, |\slshape|,
-|\textsc|, |\scshape|, |\textup| and |\upshape|.  By default these are
-|it|, |sl|, |sc| and |n|.
+|\textsc|, |\scshape|, |\textssc|, |\sscshape|, |\textsw|, |\swshape|,
+|\textulc|, |\ulcshape|, |\textup| and |\upshape|.  By default these are
+|it|, |sl|, |sc|, |ssc|, |sw|, |ulc| and |n|.  |ulc| is special here
+since it is a virtual one; it doesn't exist as a real shape.  The
+respective macros |\textulc| or |\ulcshape| change small capitals back
+to upper/lower case which in turn will not change the font with respect
+to italics, slanted or swash.  |\upshape| or |\textup| behave
+differently here and switch italics or slanted back to upright.
```

Comment:
\upshape or textup in contrast will switch italics or slanted back to upright but to not alter the state of upper/lower case, e.g., keep small capitals if present.